spec/DataSource.jest.ts

```diff
@@ -117,7 +117,7 @@ describe('GenericDatasource', () => {
     templateSrvStub.replace = (data) => data ?? '';
     setTemplateSrv(templateSrvStub);
 
-    ds.listMetrics('search').then((result) => {
+    ds.searchMetrics('search').then((result) => {
```

Comment:
This can be reverted if we do this https://github.com/simPod/GrafanaJsonDatasource/pull/324/files#r965894312